File: 900/CF96A_Football.java
import java.util.Scanner;
public class CF96A_Football 
{
    public static void main(String[] args) 
    {
        Scanner sc=new Scanner(System.in);
        String players=sc.nextLine();

        int c=0;
        int key=players.charAt(0);

        if (players.length()<7)
        {
            System.out.println("NO");
            System.exit(0);
        }

        for (int i=0;i<players.length();i++)
        {
            if (players.charAt(i)==key)
            {
                c++;
            }
            else
            {
                key=players.charAt(i);
                c=1;
            }
            if (c==7)
            {
                System.out.println("YES");
                System.exit(0);
            }
        }
        System.out.println("NO");
        sc.close();
    }   
}

File: 900/CF96A_Football.py
players=input()
key=players[0]
c=0

if len(players)<7:
    print("NO")
    exit(0)

for i in range(0,len(players)):
    if players[i]==key:
        c+=1
    else:
        key=players[i]
        c=1
    
    if c==7:
        print("YES")
        exit(0)

print("NO")